tests/h/activity/bucketing_test.py

```diff
@@ -24,9 +24,9 @@ def __eq__(self, timeframe):
             and self.document_buckets == timeframe.document_buckets
         )
 
-    def __repr__(self):
-        # pragma: no cover
-        return f'{self.__class__} "{self.label}" with {len(self.document_buckets)} document buckets'
+    # pragma: nocover
+    def __repr__(self):  # pragma: nocover
+        return f'{self.__class__} "{self.label}" with {len(self.document_buckets)} document buckets'  # pragma: nocover
```

Comment:
I've no idea which of these are doing what. I had real trouble getting this to stop being counted

Reply:
Needs more no cover